vswitchd/bridge.c

```diff
@@ -3547,6 +3746,18 @@ bridge_create(const struct ovsrec_bridge *br_cfg)
 
     hmap_init(&br->mappings);
     hmap_insert(&all_bridges, &br->node, hash_string(br->name, 0));
+
+#if defined(P4OVS)
+    /* TODO: Implement a better logic for unique bridge ID of type uint8. */
+    if (last_p4_bridge_id_used <= MAX_P4_BRIDGE_ID) {
```

Comment:
The macro "MAX_P4_BRIDGE_ID" is defined with a value of 256. The variable last_p4_bridge_id_used has a width of 8 bits. 
The code with "If" condition and associated logic within this block lead to the circulation of bridge ID values from 0 to 255 continuously, is this intentional, or do you intend to restrict it to the range of 0 to 255 only? 
With the existing logic, there is a high chance of reusing a bridge ID that is already allocated and in use. Please review the logic once again and ensure that it functions as intended.

Reply:
Currently we are rstricting the number of bridges to 256. Once bridge ID value reaches 256, we cannot allocate any more bridges which can be used by p4 target. We need a better logic here. Will take it as an HSD.